code/modules/antagonists/eldritch_cult/eldritch_antag.dm

```diff
@@ -6,6 +6,7 @@
 	job_rank = ROLE_HERETIC
 	antag_hud_type = ANTAG_HUD_HERETIC
 	antag_hud_name = "heretic"
+	can_hijack = HIJACK_HIJACKER
```

Comment:
should we give eldritch monsters the ability to hijack as well?

Reply:
good idea